pandas/tests/reshape/merge/test_merge.py

```diff
@@ -431,11 +424,11 @@ def test_merge_nosort(self):
              "var2": np.random.randint(0, 10, size=10),
              "var3": [datetime(2012, 1, 12), datetime(2011, 2, 4),
                       datetime(
```

Comment:
you can put *all* of the datetime on separate lines to avoid this kind of splitting